ZitiPacketTunnel/ViewController.swift

```diff
@@ -186,7 +186,7 @@ class ViewController: NSViewController, NSTextFieldDelegate {
             
             idLabel.stringValue = zId.id
             idNameLabel.stringValue = zId.name
-            idNetworkLabel.stringValue = zId.czid?.ztAPI ?? ""
+            idNetworkLabel.stringValue = zId.czid?.ztAPIs?.joined(separator: ", ") ?? zId.czid?.ztAPI ?? ""
```

Comment:
@smilindave26 fyi this blows out the width of the app window. I tried using newlines in the separator but the constraints didn't respect them (so the `Controller Version` value was obscured by the controller list for ha networks). I'm looking into better constraints but let me know if it's not worth the trouble at least for now.

Reply:
For now, let's just set it to the first one on the list.  Eventually, we'll need to revisit (e.g., make it a list control and update things like the controller version when changing. There are other things in that part of the UI that we improve at some point soon...

Reply:
controllerVersion comes from a call to the C SDK, passing in a ztx.  So we might end up with a version from a controller that's not being displayed.  Easy to make the single displayed controller the one currently being used (and make sure the version correctly matches)?